test/governance/Governor.test.js

```diff
@@ -18,24 +16,47 @@ const Governor = artifacts.require('GovernorMock');
 const CallReceiver = artifacts.require('CallReceiverMock');
 
 contract('Governor', function (accounts) {
+  const helper = new GovernorHelper();
```

Comment:
By declaring the instance here it's getting shared by all test runs even though they use different Governor instances (set in `beforeEach`).

But `GovernorHelper` could be stateful and be mixing state from different runs and that would lead to issues. So I think we should either:
- define `let helper;` and then assign `helper = new GovernorHelper()` in `beforeEach`
- or assign `this.helper = new GovernorHelper()`, or even `this.g = new GovernorHelper()` to make it nice and short?